server/routes/routes.js

```diff
@@ -24,4 +28,64 @@ router.get("/activities", async (req, res) => {
     res.status(500).send("Failed to fetch data");
   }
 });
+
+router.post("/create-user", async (req, res) => {
```

Comment:
/user on POST will denote that is a create user operation. please rename it

Reply:
do you mean rename to something like /register?